images/build/Dockerfiles/gitHubActions.Dockerfile

```diff
@@ -58,7 +58,16 @@ RUN apt-get update \
     # This is the folder containing 'links' to benv and build script generator
     && mkdir -p /opt/oryx
 
-RUN if [ "${DEBIAN_FLAVOR}" = "bullseye" ]; then \
+RUN if [ "${DEBIAN_FLAVOR}" = "bookworm" ]; then \
+        apt-get update \
+        && apt-get install -y --no-install-recommends \
```

Comment:
Are these packages required for all platforms? If not, they can probably be dynamically installed? - [Example](https://github.com/microsoft/Oryx/blob/main/src/BuildScriptGenerator/DotNetCore/DotNetCorePlatformInstaller.cs#L50)

Reply:
These dependencies are not limited to .NET